Examples/Ising1d/ising1d.py

```diff
@@ -12,33 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+
+from jax import numpy as jnp
 import netket as nk
 
 # 1D Lattice
-g = nk.graph.Hypercube(length=20, n_dim=1, pbc=True)
+L = 20
+g = nk.graph.Hypercube(length=L, n_dim=1, pbc=True)
 
 # Hilbert space of spins on the graph
 hi = nk.hilbert.Spin(s=1 / 2, N=g.n_nodes)
 
 # Ising spin hamiltonian
-ha = nk.operator.Ising(h=1.0, hilbert=hi, graph=g)
+ha = nk.operator.Ising(hilbert=hi, graph=g, h=1.0)
 
 # RBM Spin Machine
-ma = nk.machine.RbmSpin(alpha=1, hilbert=hi)
-ma.init_random_parameters(seed=1234, sigma=0.01)
-
+ma = nk.models.RBM(alpha=1, dtype=jnp.float64)
```

Comment:
can we make it default to 64 bits and remove jnp?

Reply:
ah, i see. 
Ok

Reply:
so you want our stuff to default to double precision?